plutus-tx-plugin/test/Plugin/Data/recursive/interListConstruct.tplc-read.golden

```diff
@@ -0,0 +1,265 @@
+(program 1.1.0 ((/\(InterList_i0 :: * -> * -> *) ->
```

Comment:
Hmm, I feel like this would be much better if we broke after the version. Not sure why we aren't?

Reply:
Because in most cases we don't do anything to pretty-print things prettily.